src/main/java/org/jboss/xavier/integrations/jpa/service/AnalysisService.java

```diff
@@ -28,6 +31,9 @@
     @Autowired
     AnalysisRepository analysisRepository;
 
+    @Value("${pagination.report.limit.max:1000}")
```

Comment:
I think we can reuse the global variable https://github.com/project-xavier/xavier-integration/blob/e4293da7eac386a3a82f5ad611a851aebce25daf/src/main/resources/application.properties#L165

Reply:
Well in fact no.... it's the route *to-pagebean* who takes care of it.
I have already modified the code adding you latest changes about pagination, to this PR.